browser/views.py

```diff
@@ -220,24 +221,43 @@ def _handle_known_person_form(request, extra_form, person):
 
 
 @staff_member_required
-def edit_person(request, person_id):
-    person = get_object_or_404(Person, pk=person_id)
+def edit_person(request, person_id=None):
```

Comment:
let's call that method something that is true for both edit and add

Reply:
Changed method name to handle_person